setup/docker/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Mount application folder to Apache default location.
+# This avoids creating and configuring a new Apache location.
+if [ -d /var/www/html ]; then
+  cd /
+  rm -rf /var/www/html
+fi
+ln -s ${APP_HOME}/src /var/www/html
+
+# Install dependencies
+cd ${APP_HOME}
+chown -R www-data:www-data src /var/www
+npm install
```

Comment:
Unless I'm wrong, this one is still run with root account. I saw a warning appearing in the console about a failed "npm install" run, not sure it's this one, but it's most likely prevent the website from working anyway. I'll try to troubleshoot exactly where it is next time I manage to get everything (VM + Docker) working 🤒

Reply:
Can you share me the log?
I can help figure out what's wrong and fix it.
This npm install should be able to run with any user, including root.